biopet-utils/pom.xml

```diff
@@ -31,6 +31,11 @@
 
     <dependencies>
         <dependency>
+            <groupId>com.typesafe.play</groupId>
```

Comment:
That blog post is slightly outdated, I think. Dispatch is still quite maintained (well, a newer version of it): https://github.com/dispatch/reboot. Also, if this is still not suitable, there should be a lot of smaller Java libraries for HTTP clients right? I think there is an Apache project made for this.